examples/examples_test.go

```diff
@@ -1,3 +1,4 @@
+// +build !shared_env
```

Comment:
`// +build` is deprecated and replaced with `//go:build`.

Also, the docs encourage a blank line after the declaration.

See: https://pkg.go.dev/cmd/go#hdr-Build_constraints